chapters/planning-data-work.tex

```diff
@@ -1,147 +1,457 @@
+On MacOS, this will be a folder with your username.
+On Windows, this will be something like ``This PC''.
 Ensure you know how to get the \textbf{absolute file path} for any given file.
+Using the absolute file path, starting from the filesystem root,
+means that the computer will never accidentally load the wrong file.
+  \index{file paths}
 On MacOS this will be something like \path{/users/username/dropbox/project/...},
-and on Windows, \path{C:/users/username/github/project/...}.
-We will write file paths such as \path{/Dropbox/project-title/DataWork/EncryptedData/}
-using forward slashes, and mostly use only A-Z, dash, and underscore.
-You should \textit{always} use forward slashes (\texttt{/}) in file paths,
-just like an internet address, and no matter how your computer writes them,
-because the other type will cause your work to break many systems.
-You can use spaces in names of non-technical files, but not technical ones.\sidenote{
-\url{http://www2.stat.duke.edu/~rcs46/lectures_2015/01-markdown-git/slides/naming-slides/naming-slides.pdf}}
-Making the structure of your files part of your workflow is really important,
-as is naming them correctly so you know what is where.
-
-\subsection{Folder management}
-
-The first thing your team will need to create is a shared folder.\sidenote{Common tools for folder sharing are Dropbox, Box, and OneDrive.}
-If every team member is working on their local computers,
-there will be a lot of crossed wires when collaborating on any single file,
-and e-mailing one document back and forth is not efficient.
-Your folder will contain all your project's documents.
-It will be the living memory of your work.
-The most important thing about this folder is for everyone in the team to know how to navigate it.
-Creating folders with self-explanatory names will make this a lot easier.
-Naming conventions may seem trivial,
-but often times they only make sense to whoever created them.
-It will often make sense for the person in the team who uses a folder the most to create it.
-
-For the purpose of this book,
-we're mainly interested in the folder that will store the project's data work.
-Agree with your team on a specific folder structure, and
-set it up at the beginning of the research project
-to prevent folder re-organization that may slow down your workflow and,
-more importantly, prevent your code files from running.
-DIME Analytics created and maintains
-\texttt{iefolder}\sidenote{\url{https://dimewiki.worldbank.org/wiki/iefolder}}
-as a part of our \texttt{ietoolkit} suite.
-This command sets up a standardized folder structure for what we call the \texttt{/DataWork/} folder.\sidenote{\url{https://dimewiki.worldbank.org/wiki/DataWork_Folder}}
-It includes folders for all the steps of a typical DIME project.
-However, since each project will always have its own needs,
-we tried to make it as easy as possible to adapt when that is the case.
+and on Windows, \path{C:/users/username/Github/project/...}.
```

Comment:
Why is the Mac path to DropBox and the Windows path to GutHub?

Readers new to a topic gets hung up on small differences like that wondering if they should try to read in any meaning to that.

Reply:
I changed both to "git" so that it is a generic path

717a0b861969d31f621b925db0406408c57ae781